sc/src/test/java/uk/ac/ebi/atlas/search/GeneSearchServiceTest.java

```diff
@@ -9,15 +9,18 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.mockito.junit.jupiter.MockitoSettings;
 import org.mockito.quality.Strictness;
+import uk.ac.ebi.atlas.experimentimport.admin.Op;
```

Comment:
Unused import here. To prevent these things from happening to you go to Code --> Optimise imports in Intellij every once in a while when developing. That should get rid of any unused imports.